docs/src/developer/reference/oauth.md

```diff
@@ -0,0 +1,402 @@
+
+### Refresh access token
+
+The client makes a refresh request to the token endpoint by adding the parameters shown below using the `application/x-www-form-urlencoded` format with a character encoding of UTF-8 in the HTTP request entity-body.
+
+Example request:
+
+```curl
+curl -s -X POST localhost:8080/oauth/token \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -d 'refresh_token=eyJhbGciOiJFZERTQSJ9.eyJzdWIiOiI4NzI1ZTRkNC01Njc5LTQwZGEtOTI3My03YTBkMmIwYjUwMGYifQ.59IICzGoli5nfwJ1ZwRH_b3T-lRgBrralE1EZZRtadI2eKrta0kaLIZpuMWPC2Icj6-LSEBsyYLXpxOm3cNaDw&client_id=b9e65569-aa61-462d-915d-94c8d6ef17a7&grant_type=refresh_token&client_secret=3f6fbd62835859b2bac411b2a2a2a54699ec56504ee32099748de3a762d41a2d'
+```
+
+Parameters:
+
+| Parameter       | Description                                       |
+| --------------- | ------------------------------------------------- |
+| `refresh_token` | Required. The refresh token issued to the client. |
+| `client_id`     | Required. The client identifier.                  |
+| `grant_type`    | Required. Value MUST be set to `refresh_token`.   |
+| `client_secret` | Required. The client's secret.                    |
+
+Example response:
+
+```json
+{                
+  "accessToken": "eyJhbGciOiJFZERTQSJ9.eyJhdWQiOiJleGFtcGxlLmNvbSIsImV4cCI6MS42NzcyMzc3MjkyMzAyMDU3NTJlOSwiaWF0IjoxLjY3NzIzNzcyNjIzMDIwNTc1MmU5LCJpc3MiOiJleGFtcGxlLmNvbSIsInNjb3BlIjoicmVhZDpzZWxmIiwic3ViIjoiYmE5MjFjZmYtZTVlYS00MzE1LWJlM2QtZmI2MDc1NTQzZjcwIn0.__fa8l5E2L-DNaxG5tJFLF8zf5y8lpCJI0F5MOFPdewaXbOLleTJ_eAueCjG7dB7yXNVY9Uiry_W5Mw7O19UCw",
+  "expiresIn": 300,
+  "refreshToken": "eyJhbGciOiJFZERTQSJ9.eyJzdWIiOiJjMzQwOWU4OC0wYmE5LTQ3YjgtOWQ0My03YTVmMmM3YjhlNGYifQ.Padku_6pOUIlE469cW5TELtiNtO1mZnBrRpj8CRMgNVBTck7qxInz6LyXVYfSV7soAa44202yXzI0o4IcfZiBA",
+  "tokenType": "Bearer"
+}
+```
+
+### Revoke a refresh token
+
+A refresh token can be revoked as follows:
+
+```curl
+curl -i -s -X POST localhost:8080/oauth/revoke \
+  -H "Content-Type: application/json" \
+  -d '{
+    "clientId": "31a605c5-b033-405a-ab05-f8307cf22d3f",
+    "clientSecret": "d2580e9b759eca52fdf3a21532ea5aae0e08706529a0a6e6fa4ab2a3d7b39da4",
+    "refreshToken": "eyJhbGciOiJFZERTQSJ9.eyJzdWIiOiJjMzQwOWU4OC0wYmE5LTQ3YjgtOWQ0My03YTVmMmM3YjhlNGYifQ.Padku_6pOUIlE469cW5TELtiNtO1mZnBrRpj8CRMgNVBTck7qxInz6LyXVYfSV7soAa44202yXzI0o4IcfZiBA"
+  }'
+```
+
+Parameters:
+
+| Parameter       | Description                                       |
+| --------------- | ------------------------------------------------- |
+| `clientId`     | Required. The client identifier.                  |
+| `refreshToken` | Required. The refresh token issued to the client. |
+| `clientSecret` | Required. The client's secret.                    |
+
+Example response:
+
+```http
+HTTP/1.1 200 OK
+(empty-response-body)
+```
+
+### Retrieve a list of 3rd party apps with account access
+
+Authenticated endpoint to retrieve a list of all applications that have account access via OAuth.
```

Comment:
Is this part of another RFC, or a wire-specific endpoint?

Is it on the internal or external port, same as registering new applications?